pytplot/tplot_math/split_vec.py

```diff
@@ -0,0 +1,27 @@
+import pytplot
+import pydivide
+import numpy as np
+from scipy import interpolate
+from scipy.interpolate import interp1d
+import pandas as pd
+
+#SPLIT TVAR
+#store columns of TVar into new TVars
+def split_vec(tvar,newtvars,columns):
```

Comment:
I would say as a default, if "newtvars" and "columns" are not specified, make it so every column is split, and give it a default name of "data_0", "data_1", "data_2", etc.